### DISCUSSION TITLE
Without /var/www/html

### DISCUSSION THREAD

**Discussion by Koray-Ak** [NginxProxyManager/nginx-proxy-manager#1312] [General]
Hello

I want to use this way /var/www/website/test/index.html without /var/www/html
Is it possible?

Thank you very much.

**Comment by chaptergy** [comment]
You could create a custom location `/test/index.html` and redirect that to `<your-host>/var/www/website/test/index.html`. But theoretically whatever web server you are using should only expose the `/var/www/html` directory, so it sounds like you have misconfigured your web server, which would be a major security issue.

**Reply by Koray-Ak** [reply]
Thank you very much for your return

Yes i did but it does not running this like.

![server](https://user-images.githubusercontent.com/70194785/129187020-31dc9027-99ac-4d40-b0ba-4ca1c1701167.png)

Nginx Proxy Manager : Custom Locations = 
Location : /test_github/index.html
Forwad Hostname : ipaddress/var/www/test_github/index.html

**Reply by chaptergy** [reply]
How did you configure your hosts? especially the custom location?

**Reply by Koray-Ak** [reply]
Sorry I dont understand
Do you want to mean nginx proxy menager custom location?
![nginx](https://user-images.githubusercontent.com/70194785/129215874-bc75e5e1-535b-4025-ba83-8b320cca1a41.png)

**Reply by chaptergy** [reply]
Yeah, and what is the error you receive when you try to access the website? I'm unfortunately not able to set up an infrastructure to replicate your issue.
But note, when you are doing this this will only redirect the `index.html`. So if `index.html` uses any other data, like css files and images, they will not be redirected, so they will not be able to be loaded. For that to work you would have to use `/test/` as the location and `<ip>/var/ww/html/test/` as the forward hostname, that should redirect all files from the subfolder to the other folder.

**Reply by Koray-Ak** [reply]
Thank you very much for your interested.

![x](https://user-images.githubusercontent.com/70194785/129221236-c3751c44-0f84-4773-bc2f-b921dc7c4188.png)

**Reply by chaptergy** [reply]
What acutally happens when you try to visit the url with your current configuration?  
And you do not have control over the apache web server? Because you would be able to resolve this issue much quicker by changing the configuration of your apache.

**Reply by Koray-Ak** [reply]
I have created only sites-available test.conf and added this like DocumentRoot /var/www/website/test and also ln -s for sites-enabled

**Reply by Koray-Ak** [reply]
Sorry it gives error

Not Found

The requested URL was not found on this server.
Apache/2.4.38 (Debian) Server at x.x.x.x Port 80

**Reply by chaptergy** [reply]
I'm sorry, I don't really understand your setup, it seems the apache server itself is not really set up?

**Reply by Koray-Ak** [reply]
Apache server is running i think the problem is not from apache side because when i have created /var/www/html/folder it is running
I think nginx proxy manager only allow /var/www/html
When i tried to /var/www/test
it is not running

**Reply by chaptergy** [reply]
Yeah, let's just talk via gitter or email, I think that might work much better.

**Reply by Koray-Ak** [reply]
okay thank you
